### DISCUSSION TITLE
Card structure and update

### DISCUSSION THREAD

**Discussion by tripsylon** [bangcorrupt/hacktribe#273] [Q&A] [answered]
Hello so I have Hacktribe installed a version that was available on November last year, and today I decided to upgrade to the latest version , I exported all samples and patterns and decided yo format the SD card (for a cleaner “work space “). But after the formatting i realized that I’ve lost the original electribe sampler folder and only have the hacktribe folder. Is it save to put the new  vsb just to the hacktribe-system folder for update? Thank you

**Comment by enorrmann** [comment, marked as answer]
yes, when you use the hacktribe firmware, the update firmware option reads the firmware update file from the hacktribe/system folder, and the electribe original is not used in any case

**Reply by tripsylon** [reply]
A stone fell from my heart , thank you 🙏😌

**Comment by tripsylon** [comment]
Dňa pi 28. 7. 2023 o 17:14 Emilio Norrmann ***@***.***>
napísala(a):

> yes, when you use the hacktribe firmware, the update firmware option reads
> the firmware update file from the hacktribe/system folder, and the
> electribe original is not used in any case
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/bangcorrupt/hacktribe/discussions/273#discussioncomment-6577222>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4I2GIS3XLZBT3YG2AVJIU3XSPJOBANCNFSM6AAAAAA227XL64>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
Thank you very much, is there any chance you have a vsb file of the new
update that you could share?
